### PR TITLE
fallback-scsv: allow protocol_version for unsupported versions

### PR DESCRIPTION
**Pull request by tomato42**
### Description

Allow `protocol_version` alert from server when the ClientHello advertises TLS_FALLBACK_SCSV and version unsupported by server.

### Motivation and Context

RFC 7507 allows either behaviour: `inappropriate_fallback` or `protocol_version` when the protocol version is unsupported.
Modify the test case so that either option is acceptable, as long as it is consistent

### Checklist

- [x] I have read the [CONTRIBUTING.md](https://github.com/tlsfuzzer/tlsfuzzer/blob/master/CONTRIBUTING.md) document and my PR follows [change requirements](https://github.com/tlsfuzzer/tlsfuzzer/blob/master/CONTRIBUTING.md#change-requirements) therein
- [ ] the changes are also reflected in documentation and code comments
- [ ] all new and existing tests pass (see CI results)
- [ ] [test script checklist](https://github.com/tlsfuzzer/tlsfuzzer/wiki/Test-script-checklist) was followed for new scripts
- [ ] new test script added to `tlslite-ng.json` and `tlslite-ng-random-subset.json`
- [x] new and modified scripts were ran against popular TLS implementations:
  - [x] OpenSSL
  - [ ] NSS
  - [ ] GnuTLS
- [x] required version of tlslite-ng updated in requirements.txt and README.md

- - -
This change is [Reviewable](https://reviewable.io/reviews/tlsfuzzer/tlsfuzzer/922)

### CODE REVIEW

File: scripts/test-fallback-scsv.py [hunk 1]
```diff
@@ -46,6 +46,13 @@ def help_msg():
     print("                execution of preceding expected failure probe")
     print("                usage: [-x probe-name] [-X exception], order is compulsory!")
     print(" --tls-1.3      server does support TLS 1.3")
+    print(" --min-support val Minimum TLS version supported by server.")
+    print("                When set, the script will expect that CH messages")
```
Comment (FrantisekKrenzelok) [inline scripts/test-fallback-scsv.py:50]: CH message - I presume client hello, I would consider not shortening.

**Review by FrantisekKrenzelok — APPROVED** [review]
LGTM

File: scripts/test-fallback-scsv.py (hunk 1 above)
Comment (tomato42) [reply scripts/test-fallback-scsv.py:50]: Good point, fixed

**Review by FrantisekKrenzelok — APPROVED** [review]
LGTM

**Pull request merged**